server/backend/opensearch_security_client.ts

```diff
@@ -157,6 +157,9 @@ export class SecurityClient {
       //     location="https://<your-auth-domain.com>/api/saml2/v1/sso?SAMLRequest=<some-encoded-string>"
       //     requestId="<request_id>"
       //   '
+
+      console.log('Recieved error from authinfo API', error);
```

Comment:
Please remove if this is for debugging.

Reply:
These are added for debugging, will remove once the ITs succeed.